spec/actions_manager_spec.coffee

```diff
@@ -562,7 +565,11 @@ describe 'ActionsManager', ->
 
           it 'stringifies the data before reporting', ->
             payload = @sendbeacon_spy.args[0][1]
-            expect(payload.actions[0].data).to.eql VALID_LINE_ITEM_DATA_1_STRINGIFIED
+
+            data = JSON.parse(VALID_LINE_ITEM_DATA_1_STRINGIFIED)
+            data.rpos = 0
+
+            expect(payload.actions[0].data).to.eql JSON.stringify(data)
```

Comment:
I think it would be simpler to just add `rpos:       0` under `quantity` in `VALID_LINE_ITEM_DATA_1` on line 11, and keep the test as it is.

```suggestion
            expect(payload.actions[0].data).to.eql VALID_LINE_ITEM_DATA_1_STRINGIFIED
```

Reply:
Reported position is not in the data reported by the shop, so i doesn't seem right to be included in `VALID_LINE_ITEM_DATA_1_STRINGIFIED`.